components/LandingPage/Stats.js

```diff
@@ -1,53 +1,219 @@
-import { t, jt } from 'ttag';
+import { useState, useEffect, useRef } from 'react';
+import { c, t, jt } from 'ttag';
+import { makeStyles } from '@material-ui/core/styles';
+import { animated, useSpring } from 'react-spring';
+
+import image from './images/stats.png';
+
+const useStyles = makeStyles(theme => ({
+  sectionStats: {
+    background: 'white',
+    padding: '70px 0 90px',
+
+    [theme.breakpoints.down('md')]: {
+      padding: '50px 0 60px',
+    },
+  },
+  top: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    marginBottom: 60,
+
+    [theme.breakpoints.down('sm')]: {
+      marginBottom: 35,
+    },
+  },
+  title: {
+    textAlign: 'center',
+    color: theme.palette.secondary[500],
+    marginBottom: -100,
+
+    [theme.breakpoints.up('md')]: {
+      maxWidth: process.env.LOCALE === 'en_US' ? 500 : 'unset',
+    },
+
+    [theme.breakpoints.down('sm')]: {
+      marginBottom: 20,
+    },
+
+    '& > h3': {
+      fontSize: 48,
+      fontWeight: 'bold',
+      lineHeight: 1.45,
+      marginBottom: 22,
+
+      [theme.breakpoints.down('sm')]: {
+        fontSize: 24,
+        fontWeight: 'normal',
+        marginBottom: 12,
+      },
+    },
+
+    '& > h4': {
+      fontSize: 34,
+      fontWeight: 500,
+      lineHeight: 1.45,
+
+      [theme.breakpoints.down('sm')]: {
+        fontSize: 18,
+        fontWeight: 'normal',
+      },
+    },
+  },
+  image: {
+    '& > img': {
+      [theme.breakpoints.down('md')]: {
+        width: '100%',
+      },
+    },
+  },
+  stats: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    maxWidth: 1134,
+    margin: '0 auto',
+
+    [theme.breakpoints.down('sm')]: {
+      maxWidth: 400,
+    },
+  },
+  item: {
+    display: 'flex',
```

Comment:
Not turning `item` to flexbox container seems to make its contents align better.

### Current
![image](https://user-images.githubusercontent.com/108608/102045787-de03ef80-3e14-11eb-976e-babfcec2a83d.png)


### After removing `display: flex` and related styles
![image](https://user-images.githubusercontent.com/108608/102045726-b1e86e80-3e14-11eb-8d13-b83cfa6d1cf6.png)